RELEASE_NOTES.md

```diff
@@ -223,3 +223,7 @@ v1.12.6
 v1.12.7
 ----
 - Update Appcenter library version
+
+v1.12.8
+ ----
+ - AppLovin Impression Level Ad Revenue and pull request
```

Comment:
I think you missed to remove these words ` and pull request`

Reply:
this was resolved in this commit 
[df1ff55](https://github.com/tenjin/tenjin-ios-sdk/pull/96/commits/df1ff559d5d72a9d3face88877f92ad210c7c58b)